app/models/comment.rb

```diff
@@ -3,6 +3,8 @@ class Comment < ActiveRecord::Base
   belongs_to :user
   belongs_to :application
 
+  scope :recent, -> { limit(3) }
```

Comment:
You might want to add an order to this scope if an order is not already given. Without an explicit `ORDER BY` there's no guarantee database rows will come back in a consistent order.